lib/rawQueries.js

```diff
@@ -80,7 +84,8 @@ const QUERY_MAPPINGS = {
     STATUS_QUERY: Symbol('status_query'),
     LATEST_BUILD_QUERY: Symbol('latest build'),
     DELETE_STEPS_QUERY: Symbol('delete steps'),
-    PR_JOBS_FOR_PIPELINE_SYNC: Symbol('pull request jobs for pipeline sync')
+    PR_JOBS_FOR_PIPELINE_SYNC: Symbol('pull request jobs for pipeline sync'),
+    PIPELINE_USAGE_COUNT: Symbol('number of pipelines using each template')
```

Comment:
`PIPELINE_USAGE_COUNT` is more generic. We need to specify that it is in the context of Job Templates.
```suggestion
    PIPELINE_USAGE_COUNT_FOR_JOB_TEMPLATES: Symbol('number of pipelines using each job template')
```